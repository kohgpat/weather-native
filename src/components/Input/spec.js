/*global it, expect*/
import React from "react";
import Input from "./index";
import renderer from 'react-test-renderer';

it("renders", () => {
  const input = renderer.create(<Input />).toJSON();
  expect(input).toMatchSnapshot();
});
