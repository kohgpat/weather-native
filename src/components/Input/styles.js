import styled from "styled-components/native";

export const Input = styled.TextInput`
  flex: 1;
  flex-shrink: 0;
  height: 40px;
  width: 100%;
  padding: 8px;
  font-size: 16px;
  border: 1px solid #ccc;
`;
