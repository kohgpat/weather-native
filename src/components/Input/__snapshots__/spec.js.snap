// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<TextInput
  allowFontScaling={true}
  style={
    Array [
      Object {
        "borderColor": "#ccc",
        "borderStyle": "solid",
        "borderWidth": 1,
        "fontFamily": "Roboto-Condensed-Regular",
        "fontSize": 16,
        "height": 40,
        "paddingBottom": 8,
        "paddingLeft": 8,
        "paddingRight": 8,
        "paddingTop": 8,
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
